src/lazy/expanded/macro_evaluator.rs

```diff
@@ -1466,11 +1566,12 @@ mod tests {
     #[test]
     fn flex_uint_parameters() -> IonResult<()> {
         let template_definition = "(macro int_pair (flex_uint::$x flex_uint::$y) (values $x $y)))";
+        let macro_id = MacroTable::FIRST_USER_MACRO_ID as u8;
         let tests: &[(&[u8], (u64, u64))] = &[
             // invocation+args, expected arg values
-            (&[0x04, 0x01, 0x01], (0, 0)),
-            (&[0x04, 0x09, 0x03], (4, 1)),
-            (&[0x04, 0x0B, 0x0D], (5, 6)), // TODO: non-required cardinalities
+            (&[macro_id, 0x01, 0x01], (0, 0)),
+            (&[macro_id, 0x09, 0x03], (4, 1)),
+            (&[macro_id, 0x0B, 0x0D], (5, 6)), // TODO: non-required cardinalities
```

Comment:
🗺️ Adding a new system macro changed the IDs of all the user macros defined in unit tests. Most tests were updated a while ago to compute their macro relative to the first user macro ID available, but there were a couple of tests that hadn't. This is one of them.